### ISSUE TITLE
Humanizer Type Initializer error 

### ISSUE THREAD

**Issue by MarinosC** [Humanizr/Humanizer#529]
I am using the release 1,37.7 of Humanizer. At a particular site the library is causing exceptions in all referenced modules. At this site I have installed the software on two Windows XP client PCs and a Windows Server 2008 R2.  When I run the application I get exceptions of: "the type initializer for Humanizer.StringHumanizeExtensions threw an exception".

The software is targeting .NET 4. On the Windows Server 2008 R2 for example, I noticed that both .NET 4 Client and .NET 4 extended runtime are installed. I do not experience this issue at other sites so I am not sure what the problem is.

![humanizer_issue](https://cloud.githubusercontent.com/assets/17309384/13136259/a651cba4-d620-11e5-9aea-368af121b031.png)

![installeddependancies](https://cloud.githubusercontent.com/assets/17309384/13136268/be92a594-d620-11e5-8b06-08708534a7bc.png)

**Comment by onovotny** [comment]
Closing for now w/o a repro. Please let me know if you see this with 2.0

**Comment by MarinosC** [comment]
Oren,

I think I am getting to the bottom of this issue.

I noticed that several PCs were also experiencing issues with Humanizer. Coincidentally, these machines were running Windows XP.

My application targets .Net Version 4.0 because many customers still use Windows XP. and Windows Server 2003. At the customer I mentioned in the opening thread, the operating systems are Windows Server 2008 R2 and Windows 7 Professional but I noticed that automatic updates were turned-off.

Last week, we enabled updates on these machines and after these were applied, the problem went away.

However no such luck with the Windows XP PCs.
